instance/models/mixins/openedx_theme.py

```diff
@@ -150,6 +158,12 @@ def get_v1_theme_settings(self):
                 'value': value
             })
 
+        if application.hero_cover_image:
+            sass_overrides.append({
+                'variable': 'homepage-bg-image',
```

Comment:
Is this variable used in simple-theme?
I see only `$home-page-hero-title-color` and `$home-page-hero-subtitle-color` in the [README](https://github.com/open-craft/edx-simple-theme/tree/simple-theme-enhancements).

Reply:
@Agrendalath, this is a built-in SASS variable already in the platform.